compiler/testData/codegen/box/extensionFunctions/contextReceivers/dp.kt

```diff
@@ -0,0 +1,18 @@
+// TARGET_BACKEND: JVM_IR
```

Comment:
Please, add irText tests for all basic constructs that use context receivers.

Reply:
Also, please, add bytecodeListing tests for declarations with context receivers. Currently these tests will most likely show incorrect method signatures, but that's Ok, we'll fix them later - it's just important for us to know that we have all valid cases covered by tests.